subsys/bluetooth/mesh/rpr_cli.c

```diff
@@ -224,6 +229,8 @@ static int handle_pdu_outbound_report(struct bt_mesh_model *mod, struct bt_mesh_
 		return 0;
 	}
 
+	k_work_reschedule(&cli->link.timeout, K_SECONDS(cli->link.time));
```

Comment:
Shouldn't PDU Outbound report with wrong OutboundPDUnumber restart link timeout either? Destination is correct in this case.